IdentityCore/src/broker_operation/request/token_request/MSIDBrokerOperationBrowserTokenRequest.h

```diff
@@ -38,15 +37,13 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly) NSDictionary *headers;
 @property (nonatomic, readonly) NSData *httpBody;
 @property (nonatomic, readonly) BOOL useSSOCookieFallback;
-@property (nonatomic, readonly) MSIDExternalSSOContext *ssoContext;
```

Comment:
Looks like my changes got reverted as well

Reply:
Thanks, I looked into primary selection logic otherwise, and it looked good.
I'll do a final review pass and approve after those changes are reverted.

Reply:
Can you take a look at https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/pull/1213 instead, it has changes from this PR and latest dev.  I'll abandon this PR